examples/getting-started/basic-setup/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "getting-started-basic-setup",
+  "version": "1.0.0",
+  "license": "MIT",
+  "private": true,
+  "author": "",
+  "scripts": {
+    "start": "serve ."
+  },
+  "keywords": [
+    "webpack",
+    "documentation",
+    "examples"
+  ],
+  "dependencies": {
+    "lodash": "latest"
+  },
+  "devDependencies": {
+    "webpack": "latest",
+    "webpack-cli": "latest",
+    "serve": "latest"
```

Comment:
This is not something you would end with by following the guide. `lastest` should be replaced by `^x.x.x`. It's fine to not commit a package-lock.json here so when running the example it actually uses the lastest versions according to the range.

Reply:
Would it be ok to add a `package-lock.json` though? This would make the boot process much faster because it will skip resolution, which is always a little slow.

Reply:
If so, what version of webpack would you use then? Just `^5`?

Reply:
> Would it be ok to add a `package-lock.json` though?

That would be fine for me too.

> If so, what version of webpack would you use then? Just `^5`?

I would use something real. So run `npm install webpack` and use that that gives. `^5.38.1`.
It's not unlikely that some users are copying these files/lines and they should not end up with something weird that we did only for the example.

Reply:
Ok, that makes sense. I ll change it 👍

Reply:
Updated.